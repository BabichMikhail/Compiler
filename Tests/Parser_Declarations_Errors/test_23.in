label a;
type a = integer;