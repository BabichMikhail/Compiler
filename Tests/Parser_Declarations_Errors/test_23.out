Error: Duplicate Identifier "a". Line 2 Column 6
